module/parser/src/commonMain/kotlin/org/cru/godtools/tool/model/Button.kt

```diff
@@ -29,6 +33,8 @@ class Button : Content, Styles, HasAnalyticsEvents, Clickable {
     internal companion object {
         internal const val XML_BUTTON = "button"
 
+        private val DEFAULT_GRAVITY = Gravity.Horizontal.CENTER
+        private val DEFAULT_WIDTH = Dimension.Percent(1f)
```

Comment:
Should these not be `const`?

Reply:
they can't be `const`, `const` only works for primitive types & strings